README.md

```diff
@@ -28,14 +28,17 @@ Andromeda allow:
 
 The idea on making Andromeda possible came by [Jellyfish Insights](http://jellyfishinsights.com/)
```

Comment:
Maybe we should change this to:
Andromeda was developed as part of the [Jellyfish Insights](http://jellyfishinsights.com/) and supported by the Foundation for Economic Education (FEE).

Reply:
I suggest we add BA here too.

Reply:
I suggest we add BA here too.